tests/validator/test_v1_validator.py

```diff
@@ -0,0 +1,158 @@
+import pytest
+
+from great_expectations.core.batch_config import BatchConfig
+from great_expectations.core.expectation_configuration import ExpectationConfiguration
+from great_expectations.core.expectation_suite import ExpectationSuite
+from great_expectations.data_context.data_context.abstract_data_context import (
+    AbstractDataContext,
+)
+from great_expectations.expectations.core.expect_column_values_to_be_between import (
+    ExpectColumnValuesToBeBetween,
+)
+from great_expectations.expectations.core.expect_column_values_to_be_in_set import (
+    ExpectColumnValuesToBeInSet,
+)
+from great_expectations.expectations.expectation import Expectation
+from great_expectations.validator.v1_validator import ResultFormat, Validator
+
+
+@pytest.fixture
+def context(
+    fds_data_context,
+) -> AbstractDataContext:
+    return fds_data_context
+
+
+@pytest.fixture
+def failing_expectation() -> Expectation:
+    return ExpectColumnValuesToBeInSet(
+        ExpectationConfiguration(
+            "expect_column_values_to_be_in_set",
+            kwargs={
+                "column": "event_type",
+                "value_set": ["start", "stop"],
+            },
+        )
+    )
+
+
+@pytest.fixture
+def passing_expectation() -> Expectation:
+    return ExpectColumnValuesToBeBetween(
+        ExpectationConfiguration(
+            "expect_column_values_to_be_between",
+            kwargs={
+                "column": "id",
+                "min_value": -1,
+                "max_value": 1000000,
+            },
+        )
+    )
+
+
+@pytest.fixture
+def expectation_suite(
+    failing_expectation: Expectation, passing_expectation: Expectation
+) -> ExpectationSuite:
+    suite = ExpectationSuite("test_suite")
+    suite.add_expectation(failing_expectation.configuration)
+    suite.add_expectation(passing_expectation.configuration)
+    return suite
+
+
+@pytest.fixture
+def batch_config(context: AbstractDataContext) -> BatchConfig:
+    datasource = context.get_datasource("sqlite_datasource")
+    return datasource.get_asset("trip_asset")
+
+
+@pytest.fixture
+def validator(context: AbstractDataContext, batch_config: BatchConfig) -> Validator:
+    return Validator(
+        context=context,
+        batch_config=batch_config,
+        batch_asset_options=None,
+        result_format=ResultFormat.SUMMARY,
+    )
+
+
+@pytest.mark.big
```

Comment:
Hmm why are these `big`? Seem pretty lightweight to me.

Reply:
I don't have a good answer there. TBH I was copying the pytest markers from `test_validator`, and I think it used `big` everywhere. I suspected because we actually perform a validation against a sqlite datasource on disk. Should I change it to something else (and if so what?)

Reply:
I think `unit` is appropriate - you can read about our markers in `pyproject.toml`

```
 "big: mark tests that don't have external dependencies but aren't unit tests",
```

Reply:
Oh thanks! didn't know we documented the markers there. Perfect! Should I actually be using `external_sqldialect`, as I'm using [this fixture](https://github.com/great-expectations/great_expectations/blob/develop/tests/conftest.py#L516)?

Reply:
hmm I'm actually not too sure here - could go either way I'll defer to you